Control coverage/Feature/VirtualMachine.md

```diff
@@ -1385,3 +1386,73 @@ Open remote management ports expose a VM/compute node to a high level of risk fr
 
 ___ 
 
+## Azure_VirtualMachine_DP_Use_Secure_TLS_Version_Trial 
```

Comment:
Can you check the comments for HybridCompute for VirtualMachine changes as well.

Reply:
I think all the comments are applicable for VirtualMachine changes as well.